content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,536 +1,502 @@
 ---
 reviewers:
-- mikedanese
 - luxas
-- errordeveloper
-- jbeda
-title: Creating HA clusters with kubeadm
+- timothysc
+- detiber
+- chuckha
+title: Creating Highly Available Clusters with kubeadm
 content_template: templates/task
 ---
 
 {{% capture overview %}}
 
-This guide shows you how to install and set up a highly available Kubernetes cluster using kubeadm.
+This page explains two different ways to set up a highly available Kubernetes
+cluster using kubeadm:
 
-This document shows you how to perform setup tasks that kubeadm doesn't perform: provision hardware; configure multiple systems; and load balancing.
+- With stacked masters. This approach requires less infrastructure. etcd members
+and control plane nodes are co-located.
+- With an external etcd cluster. This approach requires more infrastructure. The
+control plane nodes and etcd members are separated.
+
+Your clusters must run Kubernetes version 1.11 or later.
```

Comment:
@chuckha @Bradamant3 I think we need to reword this, but I'm failing on thinking of a better way to say that this doc will only work for v1.11 and later.

Also, should this be highlighted as a note to draw more attention to this?

Reply:
I haven't come up with anything better either, but I'll keep working on it.

Reply:
Maybe a "before you begin" section like this one? https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/